### PR TITLE
Added componets and Redux

### PR DESCRIPTION
**Pull request by dmitrii-casapiuc**

### CODE REVIEW

File: node/src/store/actions/products.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import {FETCH_PRODUCTS_SUCCESS} from "./actionsTypes";
+import axios from 'axios'
+
+export function fetchProducts() {
+    return async dispatch => {
+
+        try {
+            const response = await axios.get('http://192.168.99.100/api/products')
```
Comment (WictorT) [inline node/src/store/actions/products.js]: extract address to an environment variable

File: node/src/store/actions/products.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            const products = response.data.data
+
+            dispatch(fetchTodosSuccess(products))
+        } catch (e) {
```
Comment (WictorT) [inline node/src/store/actions/products.js]: remove try/catch

File: node/src/index.js [hunk 3]
```diff
@@ -1,5 +1,33 @@
...
+import {Provider} from 'react-redux'
+import thunk from 'redux-thunk'
+import {BrowserRouter} from 'react-router-dom'
+
 import App from './App';
+import './styles.css'
+import rootReducer from "./store/reducers/rootReducer";
+
+const composeEnhancers =
+    typeof window === 'object' &&
```
Comment (WictorT) [inline node/src/index.js:13]: adjust code syntax and enable only in case of prod environment

File: node/src/store/actions/actionsTypes.js [hunk 4]
```diff
@@ -0,0 +1 @@
+export const FETCH_PRODUCTS_SUCCESS = 'FETCH_PRODUCTS_SUCCESS'
```
Comment (WictorT) [inline node/src/store/actions/actionsTypes.js]: Add new line at the end of the file. Do this everywhere where needed

File: node/src/store/actions/products.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+            console.log(e)
+        }
+    }
+}
+
+export function fetchTodosSuccess(products) {
```
Comment (WictorT) [inline node/src/store/actions/products.js]: fix namming

**Review by WictorT — CHANGES_REQUESTED** [review]

File: node/src/store/actions/products.js (hunk 1 above)
Comment (dmitrii-casapiuc) [reply node/src/store/actions/products.js]: Done

File: node/src/store/actions/products.js (hunk 2 above)
Comment (dmitrii-casapiuc) [reply node/src/store/actions/products.js]: Done

File: node/src/store/actions/actionsTypes.js (hunk 4 above)
Comment (dmitrii-casapiuc) [reply node/src/store/actions/actionsTypes.js]: Done

File: node/src/store/actions/products.js (hunk 5 above)
Comment (dmitrii-casapiuc) [reply node/src/store/actions/products.js]: Done

File: node/src/index.js (hunk 3 above)
Comment (dmitrii-casapiuc) [reply node/src/index.js:13]: Done

File: node/src/store/actions/products.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
+import {FETCH_PRODUCTS_SUCCESS} from "./actionsTypes";
+import axios from 'axios'
+
+export function fetchProducts() {
+    return async dispatch => {
+        const host = process.env.REACT_APP_HOST
```
Comment (WictorT) [inline node/src/store/actions/products.js]: add env file/

File: node/src/store/actions/products.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+        const response = await axios.get(`${host}api/products`)
+        const products = response.data.data
+
+        console.log(process.env)
```
Comment (WictorT) [inline node/src/store/actions/products.js]: remove console logs

File: node/src/App.js [hunk 8]
```diff
@@ -1,54 +1,17 @@
-import React, {Component} from 'react';
+import React, {Component} from 'react'
+import Header from "./componets/Header/Header";
```
Comment (WictorT) [inline node/src/App.js:2]: Find a solution not to use relative paths when importing components

File: node/src/store/actions/products.js (hunk 6 above)
Comment (WictorT) [reply node/src/store/actions/products.js]: and name it API_PUBLIC_URL

File: node/src/store/actions/products.js [hunk 9]
```diff
@@ -0,0 +1,21 @@
+import {FETCH_PRODUCTS_SUCCESS} from "./actionsTypes";
+import axios from 'axios'
+
+export function fetchProducts() {
+    return async dispatch => {
+        const host = process.env.REACT_APP_HOST
+        const response = await axios.get(`${host}api/products`)
```
Comment (WictorT) [inline node/src/store/actions/products.js]: put `/` before api/products

**Review by WictorT — CHANGES_REQUESTED** [review]

File: node/src/App.js (hunk 8 above)
Comment (dmitrii-casapiuc) [reply node/src/App.js:2]: We added a new issue

**Pull request merged**